studio/data/analytics/project-log-requests-count-query.ts

```diff
@@ -0,0 +1,57 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { get, isResponseOk } from 'lib/common/fetch'
+import { API_URL } from 'lib/constants'
+import { analyticsKeys } from './keys'
+
+export type ProjectLogRequestsCountVariables = {
+  projectRef?: string
+}
+
+export type ProjectLogRequestsCountResponse = {
+  result: UsageApiCounts[]
+}
+export interface UsageApiCounts {
+  count: number
+}
+
+export async function getProjectLogRequestsCountStats(
+  { projectRef }: ProjectLogRequestsCountVariables,
+  signal?: AbortSignal
+) {
+  if (!projectRef) {
+    throw new Error('projectRef is required')
+  }
+
+  const response = await get<ProjectLogRequestsCountResponse>(
```

Comment:
let's use the new get method from data/fetchers
they're typed with the API specs

Reply:
also just to double check if this endpoint is already available on prod?

Reply:
It's on prod yes, will use the new data fetcher